FiveCalls/FiveCalls/Router.swift

```diff
@@ -0,0 +1,53 @@
+//
+//  File.swift
+//  FiveCalls
+//
+//  Created by Nick O'Neill on 10/8/23.
+//  Copyright © 2023 5calls. All rights reserved.
+//
+
+import SwiftUI
+
+class Router: ObservableObject {
+    @Published var path = NavigationPath()
+    
+    func back() {
+        path.removeLast()
+    }
+    
+    func backToRoot() {
+        path = NavigationPath()
+    }
+}
+
+struct IssueDetailNavModel {
```

Comment:
nit: these probably belong in their own file, or maybe in the view files that use them?

Reply:
👍 will move these into the views that use that model